### PR TITLE
Adding more kernel config paths

### PR DESCRIPTION
**Pull request by mitar**
**What this PR does / why we need it**:

This adds additional paths to find `.config` file which seems to work on the default Ubuntu 18.04 LTS installation. `"/boot/config-" + k.kernelRelease,` from that list already finds one combination, but this is not always available. In the case I am interested in it is not available because I am running Kubernetes inside a Docker container, so it requires to volume mount `/boot` into the container. This is one way to address this, but another is also to find in multiple other paths because they might be mounted in for other reasons. For example, `/lib/modules/` might be mounted in to give (a privileged) container access to additional kernel modules, and if Kubernetes also checks for `.config` file there, this helps.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mitar** [comment]
I signed the CLA.

**Comment by neolit123** [comment]
this spawned from this thread in /kind:
https://github.com/kubernetes-sigs/kind/issues/61

TLDR kind deploys kubeadm in a container that doesn't conform our kernel validator about location for the IKCONFIG and it throws preflight warnings.
(why we validate IKCONFIG in kubeadm is another question...)

i'm OK with these two new lines, as long as we don't have a few dozens of paths to handle all possible paths that distros "invent".

/cc @BenTheElder 
/lgtm
/kind cleanup
/ok-to-test

**Review by timothysc — APPROVED** [review]
/approve
